clients/src/main/java/org/apache/kafka/common/requests/CreateTopicsRequest.java

```diff
@@ -249,9 +245,9 @@ public CreateTopicsRequest(Struct struct, short version) {
         }
 
         this.topics = topics;
-        this.timeout = struct.getInt(TIMEOUT_KEY_NAME);
-        if (struct.hasField(VALIDATE_ONLY_KEY_NAME))
-            this.validateOnly = struct.getBoolean(VALIDATE_ONLY_KEY_NAME);
+        this.timeout = struct.get(TIMEOUT);
+        if (struct.hasField(VALIDATE_ONLY))
+            this.validateOnly = struct.get(VALIDATE_ONLY);
```

Comment:
If we add a `getOrElse`, this becomes simpler.

Reply:
Fixed.